From 7f6e4f47c9660aa46805ab99e6a9b2241358d136 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Tue, 8 Sep 2020 15:32:39 +0800
Subject: [PATCH] ASoC: es8311: fix no dapm match error

This patch fix this error:
[    0.755533] es8311 4-0018: ASoC: no dapm match for DMIC MUX --> FROM ADC OUT --> SDP OUT MUX
[    0.755613] es8311 4-0018: ASoC: Failed to add route DMIC MUX -> FROM ADC OUT -> SDP OUT MUX

Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
Change-Id: I6032e9f81a79ac04ceaba590658cdfd9e539b84a
---
 sound/soc/codecs/es8311.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/es8311.c b/sound/soc/codecs/es8311.c
index 137e17140032..4627ed506907 100644
--- a/sound/soc/codecs/es8311.c
+++ b/sound/soc/codecs/es8311.c
@@ -190,7 +190,7 @@ static const struct snd_kcontrol_new es8311_dmic_mux_controls =
 	SOC_DAPM_ENUM("DMIC ROUTE", es8311_dmic_mux_enum);
 
 static const char *const es8311_adc_sdp_mux_txt[] = {
-	"FROM ADC",
+	"FROM ADC OUT",
 	"FROM EQUALIZER",
 };
 static const unsigned int es8311_adc_sdp_mux_values[] = {
-- 
2.35.3

